### PR TITLE
Update Comments/server.js

### PR DESCRIPTION
**Pull request by mudassir693**

### CODE REVIEW

File: Comments/server.js [hunk 1]
```diff
@@ -12,21 +12,26 @@ import {comments} from './comments.js'
 import {commentAction, queueConstants} from '../Common/index.js'
 let channel;
 const QueueConnection = async ()=>{
-    const rmqServer = process.env.RABBITMQ_SERVER_URL
+    const rmqServer = process.env.RABBITMQ_SERVER_URL // localhost:8909
 
-    const connection = await amqp.connect(rmqServer);
+    // const connection = await amqp.connect(rmqServer);
```
Comment (mudassir693) [inline Comments/server.js:17]: Dead Code

File: Comments/server.js [hunk 2]
```diff
@@ -12,21 +12,26 @@ import {comments} from './comments.js'
...
         const {type} = JSON.parse(data.content)
         switch(type){
-            case commentAction.POST_CREATED:
+            case commentAction.POST_CREATED: // 1
                 var {id} = JSON.parse(data.content)
                 comments[id] = []
                 channel.ack(data)
                 break
-            case commentAction.COMMENT_MODERATED:
+            case commentAction.COMMENT_MODERATED: //2
```
Comment (mudassir693) [inline Comments/server.js:34]: Magic Numbers

File: Comments/server.js [hunk 3]
```diff
@@ -12,21 +12,26 @@ import {comments} from './comments.js'
 import {commentAction, queueConstants} from '../Common/index.js'
 let channel;
 const QueueConnection = async ()=>{
-    const rmqServer = process.env.RABBITMQ_SERVER_URL
+    const rmqServer = process.env.RABBITMQ_SERVER_URL // localhost:8909
 
-    const connection = await amqp.connect(rmqServer);
+    // const connection = await amqp.connect(rmqServer);
+    // channel = await connection.createChannel();
+    // await channel.assertQueue(queueConstants.COMMENTS);
```
Comment (mudassir693) [inline Comments/server.js:19]: Dead Code

File: Comments/server.js [hunk 4]
```diff
@@ -12,21 +12,26 @@ import {comments} from './comments.js'
 import {commentAction, queueConstants} from '../Common/index.js'
 let channel;
 const QueueConnection = async ()=>{
-    const rmqServer = process.env.RABBITMQ_SERVER_URL
+    const rmqServer = process.env.RABBITMQ_SERVER_URL // localhost:8909
```
Comment (mudassir693) [inline Comments/server.js:15]: Inconsistent Naming

File: Comments/server.js [hunk 5]
```diff
@@ -12,21 +12,26 @@ import {comments} from './comments.js'
...
 
-    const connection = await amqp.connect(rmqServer);
+    // const connection = await amqp.connect(rmqServer);
+    // channel = await connection.createChannel();
```
Comment (mudassir693) [inline Comments/server.js:18]: Dead Code

File: Comments/server.js [hunk 6]
```diff
@@ -42,23 +47,26 @@ app.use(express.json())
 app.use(express.urlencoded({ extended: true }))
 // routes
 
+// app.get('/comments',(req, res)=>{
+//     return res.status(200).json({comments})
+// })
```
Comment (mudassir693) [inline Comments/server.js:52]: Dead Code

File: Comments/server.js [hunk 7]
```diff
@@ -42,23 +47,26 @@ app.use(express.json())
 app.use(express.urlencoded({ extended: true }))
 // routes
 
+// app.get('/comments',(req, res)=>{
+//     return res.status(200).json({comments})
```
Comment (mudassir693) [inline Comments/server.js:51]: Dead Code

File: Comments/server.js [hunk 8]
```diff
@@ -42,23 +47,26 @@ app.use(express.json())
 app.use(express.urlencoded({ extended: true }))
 // routes
 
+// app.get('/comments',(req, res)=>{
```
Comment (mudassir693) [inline Comments/server.js:50]: Dead Code

File: Comments/server.js [hunk 9]
```diff
@@ -12,21 +12,26 @@ import {comments} from './comments.js'
...
+    
+    let connection = await amqp.connect(rmqServer);
     channel = await connection.createChannel();
     await channel.assertQueue(queueConstants.COMMENTS);
 
     channel.consume(queueConstants.COMMENTS, (data)=>{
         const {type} = JSON.parse(data.content)
         switch(type){
-            case commentAction.POST_CREATED:
+            case commentAction.POST_CREATED: // 1
```
Comment (mudassir693) [inline Comments/server.js:29]: Magic Numbers

File: Comments/server.js [hunk 10]
```diff
@@ -12,21 +12,26 @@ import {comments} from './comments.js'
 import {commentAction, queueConstants} from '../Common/index.js'
 let channel;
 const QueueConnection = async ()=>{
```
Comment (Discoursefy) [inline Comments/server.js:14]: Dead Code - Priority: Medium

File: Comments/server.js [hunk 11]
```diff
@@ -12,21 +12,26 @@ import {comments} from './comments.js'
 import {commentAction, queueConstants} from '../Common/index.js'
 let channel;
```
Comment (Discoursefy) [inline Comments/server.js:13]: Dead Code - Priority: Medium

File: Comments/server.js [hunk 12]
```diff
@@ -12,21 +12,26 @@ import {comments} from './comments.js'
 import {commentAction, queueConstants} from '../Common/index.js'
```
Comment (Discoursefy) [inline Comments/server.js:12]: Dead Code - Priority: Medium

Comment (Discoursefy) [inline Comments/server.js:12]: Dead Code - Priority: Medium

File: Comments/server.js
Comment (Discoursefy) [inline Comments/server.js:1]: Magic Numbers

File: Comments/server.js (hunk 4 above)
Comment (Discoursefy) [inline Comments/server.js:15]: Magic Numbers

**Pull request merged**